docs/website/public/404.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Sanoma Learning Design System Documentation</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet" />
+    <link href="https://unpkg.com/prismjs@1.20.0/themes/prism-okaidia.css" rel="stylesheet" />
+    <link rel="stylesheet" href="/css/main.css" />
+  </head>
+  <body>
+    <nav class="ds-sidebar ds-flow--xs">
+  <h1 class="ds-title">
+    <a href="/">Sanoma Learning Design System</a>
+  </h1>
+
+  <div>
+  <input class="ds-search" type="search" id="search" name="search" value="" />
+</div>
+
+
```

Comment:
Is this a generated file? If so, so we check that into git?

Reply:
Yes, this one is generated (all in 'public' directory). I think we don't need to have all the generated files in git, I can change it. :)